src/fides/api/models/detection_discovery/core.py

```diff
@@ -159,6 +159,8 @@ class MonitorConfig(Base):
         "MonitorExecution",
         cascade="all, delete-orphan",
         backref="monitor_config",
+        primaryjoin="MonitorExecution.monitor_config_key == foreign(MonitorConfig.key)",
+        single_parent=True,
```

Comment:
i don't think this relationship is actually _used_ anywhere as far as i can see, so we could probably just remove it, but anyway i think this should be fine 👍